core/domain/fs_services.py

```diff
@@ -337,10 +337,10 @@ def save_original_and_compressed_versions_of_image(
 
 # TODO(#15451): Add stubs for protobuf once we have enough info regarding
 # protobuf's library.
-# The argument classifier_data_proto can accept instances of
-# `TextClassifierFrozenModel` class. But since, we excluded
-# proto_files/ from the static type annotations. This argument
-# is annotated as general object type.
+# Here we use object because the argument classifier_data_proto can accept
+# instances of `TextClassifierFrozenModel` class. But since we excluded
+# proto_files/ from the static type annotations. This argument is annotated
```

Comment:
Stubs of proto are not simple -- we have to build protobuf's stubs everytime we run MyPy because usage of protobuf can vary ( implementation of protobuf can be different according to users, so there are no universal stubs available  ). So we have to build the stubs using the main proto file. 

For that, I have already raised an idea PR -- https://github.com/oppia/oppia/pull/15601

Now, as per the plan after M1 ( mid - September), vojita going to upgrade the MyPy version and raise a PR for this ( because protobuf stubs need extra work like adding a built process )